include/exadg/incompressible_navier_stokes/preconditioners/multigrid_preconditioner_momentum.cpp

```diff
@@ -210,7 +229,8 @@ MultigridPreconditioner<dim, Number>::initialize_operator(unsigned int const lev
 
   pde_operator_level->initialize(*this->matrix_free_objects[level],
                                  *this->constraints[level],
-                                 data);
+                                 data,
+                                 this->get_mapping(this->level_info[level].h_level()));
```

Comment:
why we need the map becomes clear after looking at the initialize function below